kratos/python_scripts/multistage_analysis.py

```diff
@@ -0,0 +1,149 @@
+import importlib
+
+import KratosMultiphysics
+from KratosMultiphysics.model_parameters_factory import KratosModelParametersFactory
+
+class MultistageAnalysis(object):
```

Comment:
```suggestion
class MultistageAnalysis:
```

It is anyways derrived from `Object` ryt?

Reply:
Agree. Makes no sense as we no longer support Python 2.